data/meetups.yml

```diff
@@ -371,6 +371,17 @@ locations:
     organizers:
     - organizer: Tom Zellman
       profileImage: http://photos2.meetupstatic.com/photos/member/d/a/c/c/member_91316012.jpeg
+  - location: Columbus, Ohio
+    url: http://www.meetup.com/ember-columbus/
```

Comment:
Should be updated to http://embercolumbus.com (which currently redirects to Meetup) so that we don't have to change it later should we end up building out more content there.